### PR TITLE
Adding a new payment card component

### PR DESCRIPTION
**Pull request by jodiedoubleday**
:new:

#### What does this PR do? (please provide any background)
- Creates a payment car component
- Most ecommerce websites require logos for card or payment types they accept.
- Utilizes a image sprite of all card types at 56K

#### What tests does this PR have?
- Mocha tests to check a class is added for each card type

#### How can be this tested?
- Run `nom test` on the branch
- Manually you can run `grunt docs` and edit the code (code tab) to pass in any card type.

#### Screenshots / Screencast
screen shot 2015-07-16 at 11 22 49

#### What gif best describes how you feel about this work?
money

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Software Engineer or Developer review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

#### Software Engineer or project guru review:
- [x] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
- [x] I’ve checked for coding anti-patterns.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve run all the tests locally and they pass.

### CODE REVIEW

**Comment by hxjamesmcneil** [comment]
Taking SE/Guru

**Comment by DuncanFenning** [comment]
Taking dev

**Comment by DuncanFenning** [comment]
All looks good to me :) :+1:

File: docs/src/Components.jsx [hunk 1]
```diff
@@ -101,6 +101,16 @@ var Components = React.createClass({
           </ul>
         </article>
 
+        <article id="payment-card">
+          <h3>Payment Cards</h3>
+          <p>Easy display what payment cards your website accepts</p>
```
Comment (Joezo) [inline docs/src/Components.jsx]: easily

**Comment by hxjamesmcneil** [comment]
Looks good +1

**Pull request merged**